README.md

```diff
@@ -434,6 +434,11 @@ The following settings are optional and allow you to further configure your clus
     "*.dkr.ecr.us-east-2.amazonaws.com",
     "*.dkr.ecr.us-west-2.amazonaws.com"
   ]
+
+  [settings.kubernetes.credential-providers.ecr-credential-provider.env-variables]
+  # The following are not used with ecr-credential-provider, but are provided for illustration
+  "KEY" = "abc123xyz"
```

Comment:
For consistency with the overall API "style", where settings tend to use fully expanded words rather than abbreviations, I'd suggest either `environment` or `environment-variables`. Either is OK but I have a slight preference for `environment`.